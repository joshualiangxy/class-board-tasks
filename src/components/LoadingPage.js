import React from 'react';

const LoadingPage = () => (
  <div className="box-layout">
    <div className="loader">
      <img className="loader__image" src="/images/loader.svg" />
      <p className="loader__text">
        This loading gif is provided by{' '}
        <a href="https://loading.io/">loading.io</a>
      </p>
    </div>
  </div>
);

export default LoadingPage;
