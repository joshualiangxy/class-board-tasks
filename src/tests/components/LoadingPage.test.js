import React from 'react';
import { shallow } from 'enzyme';
import LoadingPage from '../../components/LoadingPage';

const wrapper = shallow(<LoadingPage />);

describe('render', () => {
  it('should render LoadingPage', () => expect(wrapper).toMatchSnapshot());
});
