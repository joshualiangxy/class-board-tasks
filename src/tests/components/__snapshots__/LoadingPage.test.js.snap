// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render should render LoadingPage 1`] = `
<div
  className="loader"
>
  <img
    className="loader__image"
    src="/images/loader.gif"
  />
  <p>
    This loading gif is provided by
     
    <a
      href="https://loading.io/"
    >
      loading.io
    </a>
  </p>
</div>
`;
